src/NuGet.Core/NuGet.Packaging/Signing/AttributeUtility.cs

```diff
@@ -343,7 +343,7 @@ private static byte[] GetCertificateHash(X509Certificate2 cert, Common.HashAlgor
         }
 
         // Attribute -> Hashes
-        internal static List<KeyValuePair<Common.HashAlgorithmName, byte[]>> GetESSCertIDv2Entries(CryptographicAttributeObject attribute)
+        public static List<KeyValuePair<Common.HashAlgorithmName, byte[]>> GetESSCertIDv2Entries(CryptographicAttributeObject attribute)
```

Comment:
Since you're making it public, we should check arguments and have some basic tests.  It's OK to not block on this.

Reply:
The test could be based on https://github.com/NuGet/NuGet.Client/blob/8144fdeca466151c1516b65dc30d429562890b86/test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SignedAttributeTests.cs#L105-L106

Just verify that the list returned is the same one used to create the attribute. Really the test would be a subset of this test, IsVerify is doing the same thing, just converting it to a bool result

Reply:
Done.